Resources/public/js/views/services/ez-contenteditviewservice.js

```diff
@@ -400,6 +400,10 @@ YUI.add('ez-contenteditviewservice', function (Y) {
         _redirectToAttribute: function (attr) {
             var attrRedirectionUrl = this.get(attr);
 
+            this.set('publishRedirectionUrl');
```

Comment:
@dpobel I have to clean these attributes up somehow. They store previously set, obsolete data that affects other use cases. By resetting attributes values to `undefined` I'm able to update them later like in: https://github.com/ezsystems/PlatformUIBundle/pull/600/files#diff-f806c7b7ec53181d7b83e4396abd100aR103
I found no other way to fix that behaviour. Any suggestions are welcome.